src/index.html

```diff
@@ -1,13 +1,68 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="device-width initial-scale=1">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="page__content">
+    <main>
+      <section class="card" data-qa="card">
+        <img
+            src="https://github.com/NadieinOleh/layout_product-cards/blob/master/src/images/imac.jpeg?raw=true"
+            alt="imac"
+            class="card__imac-logo"
+        >
+
+        <div class="card__product">
+          <!-- <div class="imac"></div> -->
+          <div class="description">
+            APPLE A1419 iMac 27" Retina
+            <br> 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <div class="code">
+            Product code: 195434
+          </div>
+        </div>
+
+        <div class="card__grade">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star stars__star--no-active"></div>
+          </div>
+
+          <div class="reviews">
+            Reviews: 5
+          </div>
+        </div>
+
+        <div class="card__prices">
+          <div class="price-text">Price:</div>
+          <div class="amount">$2,199</div>
+        </div>
+
+        <div class="card__link">
+          <a
+            class="card__link-buy"
+            href="/"
+            data-qa="hover"
+          >
+          BUY
```

Comment:
pay attention to spaces, please
```suggestion
            BUY
```